### PR TITLE
Readme, PersonList

### PR DESCRIPTION
**Pull request by Dawid-Zieba**
I've created PersonList folder that contains Person's slice,saga and I corrected a little our Readme

### CODE REVIEW

**Comment by BoAneta** [comment]
Code looks good form me. Great job! We just need to establish one input style, whether we use `backticks` or "double quotation marks".

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+[Demo Movie Browser](https://mateuszpiaszczyk.github.io/MoviesBrowser/)
+
+## Used technologies, languages
+
+- React
+- JavaScript
+- Redux-Saga
+
+## Further readme it will show up soon
```
Comment (BoAneta) [inline README.md:13]: I would write:
"This readme wil be updating along with our aplication."
It sounds more pro and it is correct ;)

File: src/features/MoviesList/index.js
Comment (BoAneta) [inline src/features/MoviesList/index.js:1]: here we need to format spaces to match the main branch

File: src/index.js
Comment (BoAneta) [inline src/index.js:1]: Here you used `backticks` instead of " " like it is in main branch.

**Review by BoAneta — APPROVED** [review]
Code look great. Just few small changes, mostly you need to change `` to " " since it is like this in whole app.

File: README.md (hunk 1 above)
Comment (Dawid-Zieba) [reply README.md]: Changes adjustment

**Comment by Dawid-Zieba** [comment]
Overal Changes adjustment

**Review by MateuszPiaszczyk — APPROVED** [review]
Great Work, man ;)

**Pull request merged**